openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,6 +286,20 @@ The following is a non-normative example of an Authorization Request:
     &nonce=n-0S6_WzA2Mj HTTP/1.1
 ```
 
+The following is a non-normative example of a request object with a `create_request_uri``: 
+
+```
+{
+  "iss": "https://client.example.org",
+  "aud": "https://server.example.com",
+  "iat": 1541493724,
```

Comment:
To support things like key (type) negotiation, it is required to support a parameter `wallet_metadata` or at least `jwks`/`jwks_uri` to support JARM (JWE/JWS) and ECDH-ES secured authorization response. Since in certain profiles JARM is required, this would lift a lot of effort from the verifier since the verifier does not have to compute ephemeral keys of all the different key types a wallet might support.